ports/sciter-js/portfile.cmake

```
@@ -6,8 +6,8 @@ endif()
 
 set(VCPKG_POLICY_DLLS_WITHOUT_LIBS enabled)
 
-set(SCITER_REVISION c039754b11eb3b3b39024ba3257a28a63d581c7e)
-set(SCITER_SHA 77314cb55a01ceca39f00382e78ba65507d439b9302e97dae6f5bf10d7bc308df64d440dc97c9f90c06176121710102a75767d706972700f5b4f968e9740dc96)
+set(SCITER_REVISION 74bb891b14c45a1a6f7e13d9556330fde2c38797)
```

```suggestion
set(SCITER_REVISION ea573f13174328a662d0c4189b0e3e5d59834540)
```
Should be [5.0.2.26](https://gitlab.com/sciter-engine/sciter-js-sdk/-/commit/ea573f13174328a662d0c4189b0e3e5d59834540), not master commit.

---

but there is a missing change here in this version: https://gitlab.com/sciter-engine/sciter-js-sdk/-/commit/3a3cd390bb438f1428a8190e9278a45d01f79788 so it has to be on newer commit than version tag and websocket fix which I need too

---

Weird update level, 🤔 but acceptable for this `3a3cd390bb438f1428a8190e9278a45d01f79788`.

---

ready